toke.c

```diff
@@ -4427,15 +4427,25 @@ S_intuit_more(pTHX_ char *s, char *e)
 
     if (PL_lex_brackets)
         return TRUE;
-    if (*s == '-' && s[1] == '>' && (s[2] == '[' || s[2] == '{'))
+
+    if (    *s == '-'
```

Comment:
I know the code mostly use s `*s` when only looking at one byte, but in places like this where it's looking several bytes ahead using `s[1]` etc., I think it would be clearer to use `s[0]` for the current byte.

Reply:
I agree.  I was trying to avoid changing the code unnecessarily.  But now I feel free to